ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PaywallIcon.kt

```diff
@@ -0,0 +1,169 @@
+package com.revenuecat.purchases.ui.revenuecatui.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.ui.revenuecatui.R
+import kotlin.random.Random
+
+@Composable
+internal fun PaywallIcon(
+    icon: PaywallIconName,
+    modifier: Modifier = Modifier,
+    tintColor: Color,
+) {
+    Icon(
+        modifier = Modifier
+            .aspectRatio(1.0f)
+            .fillMaxSize()
+            .then(modifier),
+        painter = painterResource(id = icon.drawable()),
+        contentDescription = null,
+        tint = tintColor,
+    )
+}
+
+/**
+ * An icon that can be displayed inside a RevenueCat paywall.
+ */
+enum class PaywallIconName(val value: String) {
```

Comment:
Should this be `internal`?

Reply:
We need to expose this for https://linear.app/revenuecat/issue/PWL-194/icons /cc @vegaro
So we can iterate through the icons inside of `features` and verify that they're all valid.

Reply:
Sorry I read your first comment as "private", I keep forgetting that Kotlin's default is `public` and not `internal` like in Swift.